rpm_spec/qubes-core-admin-client.spec.in

```diff
@@ -17,6 +17,7 @@ BuildRequires:	python%{python3_pkgversion}-xcffib
 Requires:   python%{python3_pkgversion}-qubesadmin
 Requires:   python%{python3_pkgversion}-yaml
 Requires:   qubes-repo-templates
+Requires:   qubes-rpm-oxide > 0.2.2
```

Comment:
Generally, better use `>= (version that is new enough)` than `> (version that is too old)`. Same in `debian/control`.
But for the purpose of running tests, please move it now to the PR description and add as a commit only when such version will actually exists (I can commit this change too).

Reply:
To clarify: keep (adding) the dependency, just drop the version.